docs/user-guide/get-started.md

```diff
@@ -1,13 +1,11 @@
 # Getting started
 
-Stylelint is designed for CSS.
+You can lint:
 
-However, it can be used with [PostCSS syntaxes](https://github.com/postcss/postcss#syntaxes) that:
+- CSS files by using our standard config
+- everything else by using extensions written by the community
 
-- parse CSS-like languages like SCSS, Less and SugarSS
-- extract styles from HTML, JavaScript and Markdown
```

Comment:
It now:

- maps onto two headings below: linting CSS files, and linting everything else
- establishes official and community split early on, i.e. CSS out of the box and community extensions for everything else
- avoids introducing PostCSS syntaxes (and linking off-site) so early on as can be confusing - can still be reached but via the docs for `customSyntax`